keyboards/runner3680/5x6/config.h

```diff
@@ -27,7 +27,7 @@
 /* key matrix size */
 // Rows are doubled-up
 #define MATRIX_ROWS 10
-#define MATRIX_COLS 8
+#define MATRIX_COLS 12
```

Comment:
I believe this should be:

```suggestion
#define MATRIX_COLS 6
```

Reply:
Thank you.